src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -642,61 +643,32 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
   const renderShowScheduleToggle = () => (
     <div className="toggle-room">
       <h4 className="italic input-header">Show Schedule</h4>
-      <label id={"showSchedule"} className="switch">
-        <input
-          type="checkbox"
-          id={"showSchedule"}
-          name={"showSchedule"}
-          ref={register}
-          defaultChecked={DEFAULT_SHOW_SCHEDULE}
-        />
-        <span className="slider round"></span>
-      </label>
+      <Toggler
+        id="showSchedule"
+        forwardedRef={register}
+        defaultToggled={DEFAULT_SHOW_SCHEDULE}
+      />
```

Comment:
There's no need for an `id`, because `Checkbox` nests it's `input` within the `label`, so they are inherently connected without needing the `id`.

```suggestion
      <Toggler
        name="showSchedule"
        defaultToggled={DEFAULT_SHOW_SCHEDULE}
        forwardedRef={register}
      />
```

Reply:
I'll only add this on the first of these, but for all of the `Toggler`s in this file, can we also move the label of it (eg. `<h4 className="italic input-header">Show Schedule</h4>` into `Toggler`'s `label` prop? Potentially without the `h4`.

Reply:
Would prefer subsequent PR dealing with labels due to possible styling issues. More at https://github.com/sparkletown/sparkle/pull/1775/files#r667330479

Reply:
See my thoughts on that in https://github.com/sparkletown/sparkle/pull/1775/files#r667331265

**tl;dr** Let's see how hard it will be to fix here, and if we can do it all at once, then we should. As that is really part of converting them to use the `Toggler` component.